### PR TITLE
Mgmt after Release 2.0.0 beta.2

### PR DESCRIPTION
**Pull request by ChenTanyi**

### CODE REVIEW

**Comment by ChenTanyi** [comment]
The beta.2 will make release plugin error.

**Comment by ChenTanyi** [comment]
Do a cherry-pick and force-push the last commit.

File: sdk/resources/mgmt/src/main/java/com/azure/resourcemanager/AzureServiceClient.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public String userAgent() {
     private static final String OS_NAME;
     private static final String OS_VERSION;
     private static final String JAVA_VERSION;
-    private static final String SDK_VERSION = "2.0.0-beta.2";
+    private static final String SDK_VERSION = "2.0.0-SNAPSHOT";
```
Comment (ChenTanyi) [inline sdk/resources/mgmt/src/main/java/com/azure/resourcemanager/AzureServiceClient.java:67]: @weidongxu-microsoft @xccc-msft @yungezz Do you think it need to change?

Comment (yungezz) [reply sdk/resources/mgmt/src/main/java/com/azure/resourcemanager/AzureServiceClient.java:67]: vote for change to SNAPSHOT

**Pull request merged**